crates/web_server/src/lib.rs

```diff
@@ -37,39 +45,43 @@ struct WebServerState<KEY> {
     latest_quorum_proposal: u64,
     /// view for the most recent view sync proposal
     latest_view_sync_proposal: u64,
-
-    /// view for teh oldest DA certificate
+    /// view for the oldest DA certificate
     oldest_certificate: u64,
-
+    /// view for the oldest view sync certificate
     oldest_view_sync_proposal: u64,
     /// view number -> Vec(index, vote)
     votes: HashMap<u64, Vec<(u64, Vec<u8>)>>,
-
+    /// view sync: view number -> Vec(index, vote)
     view_sync_votes: HashMap<u64, Vec<(u64, Vec<u8>)>>,
     /// view number -> highest vote index for that view number
     vote_index: HashMap<u64, u64>,
-
+    /// view_sync: view number -> highest vote index for that view number
     view_sync_vote_index: HashMap<u64, u64>,
     /// view number of oldest votes in memory
     oldest_vote: u64,
-
+    /// view sync: view number of oldest votes in memory
     oldest_view_sync_vote: u64,
-
+    /// view number -> (secret, proposal)
+    /// TODO is this right?
```

Comment:
It doesn't look like this string is actually used for anything in the logic?  I'm also confused.  Maybe just put "string" instead of proposal.  I think that is fine for now since we're getting rid of this code soon anyway.